addon/components/logo-link.js

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import layout from '../templates/components/logo-link';
+import {getOwner} from '@ember/application';
+import {computed} from '@ember/object';
+
+export default Component.extend({
+  layout,
+  siteSlug: computed(function() {
```

Comment:
the template for this component (`addon/templates/components/logo-link.hbs`) doesn't use this `siteSlug` property, so you can remove this computed property.

In fact, once you remove the `siteSlug` property, what will remain in this file is just boilerplate. Since there's no actual JS backing the template, you actually remove this entire file and ember will render the template no problem.

Reply:
If I just remove the file I get this error in the console: 

```(index):41 Uncaught Error: Could not find module `nypr-ui/components/logo-link` imported from `wnyc-web-client/components/logo-link` ```

I don't get ember's conventions yet enough to know why. Does something else need to go?

Reply:
hm, I guess I got that wrong. ignore that note and carry on then.